web/satellite/static/emails/WelcomeWithCode.html

```diff
@@ -1,185 +1,20 @@
-                          <td align="center" valign="top" width="20.00%" style="Margin:0;border:0;padding-bottom:5px;padding-top:5px;padding-right:5px;padding-left:5px;border-left:1px solid #cccccc"><a target="_blank" href="https://www.storj.io/legal/terms-of-use" rel="noopener noreferrer" style="mso-line-height-rule:exactly;text-decoration:none;font-family:-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';display:block;color:#333333;font-size:12px">Terms of Use</a></td>
-                          <td valign="top" width="20.00%" align="center" style="Margin:0;border:0;padding-bottom:5px;padding-top:5px;padding-right:5px;padding-left:5px;border-left:1px solid #cccccc"><a target="_blank" href="https://www.storj.io/legal/terms-of-service" rel="noopener noreferrer" style="mso-line-height-rule:exactly;text-decoration:none;font-family:-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';display:block;color:#333333;font-size:12px">Terms of Service</a></td>
-                          <td align="center" valign="top" width="20.00%" style="padding:0;Margin:0;border:0;padding-bottom:5px;padding-top:5px;padding-left:5px;border-left:1px solid #cccccc"><a href="https://supportdcs.storj.io/hc/en-us" rel="noopener noreferrer" target="_blank" style="mso-line-height-rule:exactly;text-decoration:none;font-family:-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';display:block;color:#333333;font-size:12px">Support</a></td>
-                         </tr>
-                       </table></td>
-                     </tr>
-                   </table></td>
-                 </tr>
-               </table></td>
-             </tr>
-           </table></td>
-         </tr>
-       </table></td>
+  <div dir="ltr" class="es-wrapper-color" lang="en" style="background-color:#FAFAFA">
+    <table cellpadding="0" cellspacing="0" align="center" class="es-content" role="none" style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;width:100%;table-layout:fixed !important">
+      <tr>
+       <td align="center" style="padding:0;Margin:0">
+        {{template "header.html" .}}
+        {{$EmailHeading := "Your 6-Digit Verification Code"}}
+        {{$EmailBody := "You're receiving this email because you signed up for a Storj account. Confirm your email address by entering the code below to get started on the distributed cloud. Do not share this code with anyone."}}
```

Comment:
note - I have not figured out how to properly pass values like these into a child template